apps/api/src/server/utils/cache-data.js

```diff
@@ -0,0 +1,14 @@
+import NodeCache from 'node-cache';
+
+export const nodeCache = new NodeCache();
+
+export const setCache = (
+	/** @type {NodeCache.Key} */ key,
+	/** @type {Record<string,any>[]} */ value
+) => {
+	nodeCache.set(key, value, 3_600_000);
```

Comment:
Can we put a comment here on how long the cache will be valid for given 3_600_000?

Reply:
we set it to 1h for now - how long do you think we should store it?

Reply:
1 hour sounds fine. I wasn't sure if this was in seconds/minutes etc, so a comment to say that on here would be great